.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,14 +23,14 @@ If user exposed functionality or configuration variables are added/changed:
   - [ ] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)
 
 If the code communicates with devices, web services, or third-party tools:
-  - [ ] [_The manifest file_][manifest-docs] has all fields filled out correctly ([example][ex-manifest]).
-  - [ ] New dependencies have been added to `requirements` in the manifest ([example][ex-requir]).
-  - [ ] New or updated dependencies have been added to `requirements_all.txt` by running `script/gen_requirements_all.py`.
-  - [ ] New files were added to `.coveragerc`.
+  - [ ] [_The manifest file_][manifest-docs] has all fields filled out correctly ([example][ex-manifest]). Update and include derived files by running `python3 -m script.hassfest`.
+  - [ ] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
+  - [ ] Python dependencies are only imported inside functions that use them ([example][ex-import]).
```

Comment:
Did we figure something out that we had missed? PR description doesn't say the reason for the revert.

Reply:
I went down the rabbit hole of links and found out myself.

There are more requirements moved than for hue I think. I've been telling contributors it's ok to import at the top now.

Reply:
Sorry, my bad for not linking it all. So I talked to Pascal about the check config add-on and we're going to move forward with installing requirements, I am going to remove this line again.